File: src/Distvisor.Web/ClientApp/src/api/models/financial-account.ts
/* tslint:disable */
/* eslint-disable */
import { FinancialAccountType } from './financial-account-type';
export interface FinancialAccount {
  name?: null | string;
  number?: null | string;
  paycards?: null | Array<string>;
  type?: FinancialAccountType;
}

File: src/Distvisor.Web/Models/FinancialAccount.cs
using System;

namespace Distvisor.Web.Models
{
    public class FinancialAccount
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Number { get; set; }
        public FinancialAccountType Type { get; set; }
        public DateTime CreatedDateTimeUtc { get; set; }
    }
}
